plugins/rsocket-transport/build.gradle

```diff
@@ -63,8 +63,14 @@ dependencies {
 
     compileOnly project(":app")
 
-    compile 'io.rsocket.rpc:rsocket-rpc-core'
-    compile 'io.rsocket:rsocket-transport-netty'
+    compile ('io.rsocket.rpc:rsocket-rpc-core') {
+        exclude group: "io.projectreactor.netty", module: "reactor-netty-core"
```

Comment:
does `rsocket-rpc-core` depend on `reactor-netty`, or it is indirect through `rsocket-transport-netty`? If the latter then this exclusion can be omitted because the next one will be taken in account

Reply:
It is not. I guess it is redundant.

Reply:
fixed